cerbero/build/recipe.py

```diff
@@ -232,7 +233,8 @@ def gen_library_file(self, output_dir=None):
                 implib = genlib.create(
                     os.path.join(self.config.prefix, dllpath),
                     self.config.target_arch,
-                    os.path.join(self.config.prefix, 'lib'))
+                    os.path.join(self.config.prefix, 'lib'),
+                    self.library_name_override)
```

Comment:
You're right.

But instead of overriding the method, what about using a dict? If the key `dllpath` exists, we send the value, else we send None.